lib/matplotlib/tests/test_colors.py

```diff
@@ -108,7 +108,7 @@ def test_double_register_builtin_cmap():
 
 
 def test_colormap_copy():
```

Comment:
This test is about the `Colormap` class; you should not be replacing it with strings.

Reply:
Strings don't even work here and are responsible for the failing tests. I believe this was an oversight. Corrections suggested below.